Fails to compile with vala 0.55.1

I maintain the `appeditor` package in Fedora Linux. Since `vala` was updated from 0.54.6 to 0.55.1 in Rawhide (the development version of Fedora), `appeditor` fails to build with the following error:

```
valac -C --define HAS_VALA040 --pkg posix --pkg gee-0.8 --pkg gtk+-3.0 --pkg granite --color=always --directory com.github.donadigo.appeditor.p --basedir ../ --gresources=../data/com.github.donadigo.appeditor.gresource.xml ../src/Application.vala ../src/MainWindow.vala ../src/AppSourceList.vala ../src/AppItem.vala ../src/CategoryItem.vala ../src/IconButton.vala ../src/SettingBox.vala ../src/PersistentPlaceholderEntry.vala ../src/FieldEntry.vala ../src/CategoryComboBox.vala ../src/AppInfoViewStack.vala ../src/AppInfoView.vala ../src/AppDirectoryScanner.vala ../src/DesktopApp.vala ../src/DesktopAppManager.vala ../src/AppInfoViewSaver.vala ../src/IconListBox.vala ../src/IconChooserDialog.vala ../src/IconRow.vala ../src/Constants.vala
../src/IconButton.vala:71.9-71.41: warning: `Granite.Widgets.Utils.set_theming' has been deprecated since 5.5.0. Use 
   71 |         Granite.Widgets.Utils.set_theming (this, css, "flat", Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                              
../src/FieldEntry.vala:55.9-55.41: warning: `Granite.Widgets.Utils.set_theming' has been deprecated since 5.5.0. Use 
   55 |         Granite.Widgets.Utils.set_theming (entry, css, Gtk.STYLE_CLASS_FLAT, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                             
../src/Application.vala:19.42-23.5: error: value is less accessible than constant `AppEditor.Application.OPTIONS'
   19 |     public const OptionEntry[] OPTIONS = {
      |                                          ^
   20 |         { "create", 'c', 0, OptionArg.FILENAME, out create_exec_filename,
      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   21 |         "Create an application entry from an executable path", "FILENAME" },
      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   22 |         { null }
      | ~~~~~~~~~~~~~~~~
   23 |     };
      | ~~~~~ 
```

I haven’t investigated closely enough to be sure if this is the *best* solution, but changing

```vala
    private static string? create_exec_filename;
```

to

```vala
    public static string? create_exec_filename;
```

in `src/Application.vala` is enough to fix the build.

I can confirm the same issue on Ubuntu/Lubuntu 22.04.1 x86_64. The package manager version of `valac` is 0.56.0-1.

```
$ meson build
The Meson build system
Version: 0.61.2
Source dir: /home/user/Development/appeditor/code
Build dir: /home/user/Development/appeditor/code/build
Build type: native build
Project name: com.github.donadigo.appeditor
Project version: 1.1.4
C compiler for the host machine: cc (gcc 11.2.0 "cc (Ubuntu 11.2.0-19ubuntu1) 11.2.0")
C linker for the host machine: cc ld.bfd 2.38
Vala compiler for the host machine: valac (valac 0.56.0)
Host machine cpu family: x86_64
Host machine cpu: x86_64
Configuring Config.vala using configuration
Found pkg-config: /usr/bin/pkg-config (1.8.0)
Run-time dependency granite found: YES 6.2.0
Run-time dependency gtk+-3.0 found: YES 3.24.33
Run-time dependency gee-0.8 found: YES 0.20.5
Library posix found: YES
Found pkg-config: /usr/bin/pkg-config (1.8.0)
Program glib-compile-resources found: YES (/usr/bin/glib-compile-resources)
Build targets in project: 21

Found ninja-1.10.1 at /usr/bin/ninja

$ cd build

$ meson configure -Dprefix=~/.localinstall

$ ninja
[0/1] Regenerating build files.
The Meson build system
Version: 0.61.2
Source dir: /home/user/Development/appeditor/code
Build dir: /home/user/Development/appeditor/code/build
Build type: native build
Project name: com.github.donadigo.appeditor
Project version: 1.1.4
C compiler for the host machine: cc (gcc 11.2.0 "cc (Ubuntu 11.2.0-19ubuntu1) 11.2.0")
C linker for the host machine: cc ld.bfd 2.38
Vala compiler for the host machine: valac (valac 0.56.0)
Host machine cpu family: x86_64
Host machine cpu: x86_64
Configuring Config.vala using configuration
Dependency granite found: YES 6.2.0 (cached)
Dependency gtk+-3.0 found: YES 3.24.33 (cached)
Dependency gee-0.8 found: YES 0.20.5 (cached)
Library posix found: YES
Found pkg-config: /usr/bin/pkg-config (1.8.0)
Program glib-compile-resources found: YES (/usr/bin/glib-compile-resources)
Build targets in project: 21

com.github.donadigo.appeditor 1.1.4

  User defined options
    backend: ninja
    prefix : ~/.localinstall

Found ninja-1.10.1 at /usr/bin/ninja
[19/41] Compiling Vala source Config.vala ../src/Application....nChooserDialog.vala ../src/IconRow.vala ../src/Constants.vala
FAILED: com.github.donadigo.appeditor.p/build/Config.c com.github.donadigo.appeditor.p/src/Application.c com.github.donadigo.appeditor.p/src/MainWindow.c com.github.donadigo.appeditor.p/src/AppSourceList.c com.github.donadigo.appeditor.p/src/AppItem.c com.github.donadigo.appeditor.p/src/CategoryItem.c com.github.donadigo.appeditor.p/src/IconButton.c com.github.donadigo.appeditor.p/src/SettingBox.c com.github.donadigo.appeditor.p/src/PersistentPlaceholderEntry.c com.github.donadigo.appeditor.p/src/FieldEntry.c com.github.donadigo.appeditor.p/src/CategoryComboBox.c com.github.donadigo.appeditor.p/src/AppInfoViewStack.c com.github.donadigo.appeditor.p/src/AppInfoView.c com.github.donadigo.appeditor.p/src/AppDirectoryScanner.c com.github.donadigo.appeditor.p/src/DesktopApp.c com.github.donadigo.appeditor.p/src/DesktopAppManager.c com.github.donadigo.appeditor.p/src/AppInfoViewSaver.c com.github.donadigo.appeditor.p/src/IconListBox.c com.github.donadigo.appeditor.p/src/IconChooserDialog.c com.github.donadigo.appeditor.p/src/IconRow.c com.github.donadigo.appeditor.p/src/Constants.c 
valac -C --debug --debug --define HAS_VALA040 --pkg posix --pkg gee-0.8 --pkg gtk+-3.0 --pkg granite --color=always --directory com.github.donadigo.appeditor.p --basedir ../ --gresources=../data/com.github.donadigo.appeditor.gresource.xml Config.vala ../src/Application.vala ../src/MainWindow.vala ../src/AppSourceList.vala ../src/AppItem.vala ../src/CategoryItem.vala ../src/IconButton.vala ../src/SettingBox.vala ../src/PersistentPlaceholderEntry.vala ../src/FieldEntry.vala ../src/CategoryComboBox.vala ../src/AppInfoViewStack.vala ../src/AppInfoView.vala ../src/AppDirectoryScanner.vala ../src/DesktopApp.vala ../src/DesktopAppManager.vala ../src/AppInfoViewSaver.vala ../src/IconListBox.vala ../src/IconChooserDialog.vala ../src/IconRow.vala ../src/Constants.vala
../src/IconButton.vala:71.9-71.41: warning: `Granite.Widgets.Utils.set_theming' has been deprecated since 5.5.0. Use 
   71 |         Granite.Widgets.Utils.set_theming (this, css, "flat", Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                              
../src/FieldEntry.vala:55.9-55.41: warning: `Granite.Widgets.Utils.set_theming' has been deprecated since 5.5.0. Use 
   55 |         Granite.Widgets.Utils.set_theming (entry, css, Gtk.STYLE_CLASS_FLAT, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                             
../src/Application.vala:19.42-23.5: error: value is less accessible than constant `AppEditor.Application.OPTIONS'
   19 |     public const OptionEntry[] OPTIONS = {
      |                                          ^
   20 |         { "create", 'c', 0, OptionArg.FILENAME, out create_exec_filename,
      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   21 |         "Create an application entry from an executable path", "FILENAME" },
      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   22 |         { null }
      | ~~~~~~~~~~~~~~~~
   23 |     };
      | ~~~~~ 
Compilation failed: 1 error(s), 2 warning(s)
ninja: build stopped: subcommand failed.
```

Changine visibility of `create_exec_filename` to `public` allows compilation to continue.

--- a/src/Application.vala
+++ b/src/Application.vala
@@ -26,7 +26,7 @@ public class AppEditor.Application : Gtk.Application {
         return Gtk.check_version (3, 22, 0) == null;
     }
 
-    private static string? create_exec_filename;
+    public static string? create_exec_filename;
 
     private MainWindow? window = null;
 

